pydo/manager.py

```diff
@@ -158,6 +159,33 @@ def _get(self, id):
         else:
             return table_element
 
+    def _update(self, id, object_values=None):
```

Comment:
There are no tests for this new method. It would be nice to add one for:

* The normal case scenario
* For the ValueError scenario

Reply:
Added in 278740781f1f0e9e6dd96ea94d67faa2f63cf865.